Tests/ApiVideoPlayerTests/IntegrationTests/ApiVideoPlayerControllerIntegrationTests.swift

```diff
@@ -0,0 +1,115 @@
+@testable import ApiVideoPlayer
+import CoreMedia
+import XCTest
+
+@available(iOS 14.0, *)
+final class ApiVideoPlayerControllerIntegrationTests: XCTestCase {
+  private let validVideoId = "vi2G6Qr8ZVE67dWLNymk7qbc"
```

Comment:
As `validVideoId` and `invalidVideoId` are declared in the two integration test classes with the same value. Put them in a separate specific class.